pytd/writer.py

```diff
@@ -249,23 +253,30 @@ def write_dataframe(self, dataframe, table, if_exists):
             - overwrite: drop it, recreate it, and insert data.
             - append: insert data. Create if does not exist.
             - ignore: do nothing.
+
+        fmt : {'csv', 'msgpack'}, default: 'csv'
+            Format for bulk_import.
```

Comment:
Can you describe pros/cons of each option?

Reply:
Added 91db2ef